maintainers/scripts/update-hackage

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -p haskellPackages.cabal2nix -i sh
+
+# Must be run from the nixpkgs root
+
+set -eu -o pipefail
+
+exit_trap()
+{
+  local lc="$BASH_COMMAND" rc=$?
+  test $rc -eq 0 || echo "*** error $rc: $lc"
+}
+
+trap exit_trap EXIT
+
+HACKAGE_REPO="${1}"
```

Comment:
```
./update-hackage: line 16: 1: unbound variable
*** error 1: HACKAGE_REPO="${1}"
```

This should go after the check for the number of args.

Reply:
thanks, I made some additionnal changes like adding a dedicated shell.nix